### DISCUSSION TITLE
Upgrading from 3.7.13 to 3.8.x ?

### DISCUSSION THREAD

**Discussion by harry-sallinggroup** [rabbitmq/rabbitmq-server#2794] [Other] [answered]
I want to upgrade my queues to the newest version as the 3.7.x is EOL.

Not sure which exactly version should I choose? Which is the most stable one? 

What are the general recommendation to keep production up-to-date? N-1?

**Comment by harry-sallinggroup** [comment, marked as answer]
No one, really?

**Reply by michaelklishin** [reply]
Go with the most recent patch unless release notes recommend skipping it (which is extremely rare).

**Reply by harry-sallinggroup** [reply]
Thank you very much!
